core/node/rpc/get_miniblocks.go

```diff
@@ -14,14 +14,32 @@ func (s *Service) localGetMiniblocks(
 	req *connect.Request[GetMiniblocksRequest],
 	stream SyncStream,
 ) (*connect.Response[GetMiniblocksResponse], error) {
-	miniblocks, terminus, err := stream.GetMiniblocks(ctx, req.Msg.FromInclusive, req.Msg.ToExclusive)
+	toExclusive := req.Msg.ToExclusive
```

Comment:
please return error here if toExlusive <= fromInlcusive (otherwise math below can be incorrect)